app/views/order_items/_order_item.text.erb

```diff
@@ -0,0 +1 @@
+<%= sprintf("%2d x %s", order_item.quantity, order_item.menu_item.description) %>
```

Comment:
Why are you limiting the order quantity to 2 digits only ( `%2d` => first 2 digits IIRC)